contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -130,9 +130,10 @@ contract OETHVaultCore is VaultCore {
         }
 
         // Amount excluding fees
-        uint256 amountMinusFee = _calculateRedeemOutputs(_amount)[
-            wethAssetIndex
-        ];
+        // No fee for the strategist, makes it easier to do operations
+        uint256 amountMinusFee = msg.sender == strategistAddr
```

Comment:
On Base we allow either the Strategies or Governor to redeem. Do we want to allow the Governor, which is the Origin Timelock contract, also to redeem?

Reply:
I thought of that, but it's gonna complicate the code a bit (since we cannot blindly add the modifier). I guess I can just add that to this conditional statement, will make that change

Reply:
Have made this change